ntfysh_client/MainForm.cs

```diff
@@ -409,5 +411,32 @@ private void exitToolStripMenuItem1_Click(object sender, EventArgs e)
             _trueExit = true;
             Close();
         }
+
+        private async void SendNotificationMenuItem_Click(object sender, System.EventArgs e)
+        {
+            await SendMessage();
+        }
+
+        private async void notificationTopics_MouseDoubleClick(object sender, MouseEventArgs e)
+        {
+            await SendMessage();
+        }
+
+        private async Task SendMessage()
+        {
+            if (notificationTopics.SelectedItem != null && string.IsNullOrEmpty(notificationTopics.SelectedItem as string) == false)
+            {
+                var dlg = new SendMessageForm();
```

Comment:
The rest of the project uses explicit type declarations instead of `var`. For consistency, please could you update all instances of `var` with their explicit types?